### PR TITLE
War on typos Act 1

### PR DESCRIPTION
**Pull request by szepeviktor**
- [x] The contributed code is licensed under GPL v2.0
- [x] Contributor Licence Agreement (CLA) is signed
- [ ] util/update-modules has been ran - Is there generated code in the repository ❔
- [ ] Documention updated
- [ ] Test suite update

Hello!
Fixed misspellings in `bin/` with a tool called `typos`.
There may be collateral damage. Please review.

After this one gets merged I proceed with other directories 🚀

### CODE REVIEW

**Comment by szepeviktor** [comment]
🎈 Fun fact: "arugment" is the TOP typo.

**Comment by szepeviktor** [comment]
At the end `typos` could run in a CI workflow!

**Comment by szepeviktor** [comment]
@svetasmirnova Friendly ping 🏓

**Review by svetasmirnova — CHANGES_REQUESTED** [review]
You changed only tools code while Percona Toolkit auto-generates most of the tool code from modules. As a result, all your work will be wiped after we run the update-modules script. Please take a look at Contributing guide for details. Please move your changes to the proper places.

**Comment by szepeviktor** [comment]
Thank you Света!
I thought the opposite, left lib/* files alone :)

Fixing ... 🏃🏻‍♂️

**Comment by szepeviktor** [comment]
@svetasmirnova Done 🍏
I've put typo fixes and generated files in separate commits for you.

File: lib/RowChecksum.pm [hunk 1]
```diff
@@ -169,7 +169,7 @@ sub make_chunk_checksum {
    my $q     = $self->{Quoter};
 
    my %crc_args = $self->get_crc_args(%args);
-   PTDEBUG && _d("Checksum strat:", Dumper(\%crc_args));
+   PTDEBUG && _d('Checksum start:', Dumper(\%crc_args));
```
Comment (szepeviktor) [inline lib/RowChecksum.pm:172]: I am unsure here.

Comment (svetasmirnova) [reply lib/RowChecksum.pm:172]: Should be OK

**Comment by svetasmirnova** [comment]
Thank you! I added more changes, so the tests are now passed.

**Review by svetasmirnova — APPROVED** [review]

**Comment by szepeviktor** [comment]
Thank you for your commit!

I was looking for an `update-everything-everywhere.sh` script but could not find it.
Contributors would find it useful.

**Review by EvgeniyPatlan — APPROVED** [review]

**Pull request merged**